extensions/chghost-3.2.md

```diff
@@ -6,41 +6,30 @@ copyrights:
     name: "Christine Dodrill"
     period: "2013"
     email: "xena@yolo-swag.com"
+  -
+    name: "Ryan"
+    period: "2016"
+    email: "ryan@hashbang.sh"
 ---
-The chghost client capability allows a server to directly inform clients about a
-host or user change without having to send a fake quit and join. This capability
+
+The chghost client capability allows servers to directly inform clients about a
+host or user change without having to send fake quits or joins. This capability
 MUST be referred to as `chghost` at capability negotiation time.
 
-When enabled, clients will get the CHGHOST message to designate the host of a
+When enabled, clients will get the `CHGHOST` message to designate the host of a
```

Comment:
When this capability is enabled*

Specifically because the more explicit 'enable the chghost capability' note below has been removed